release/pipelines/windows-efi-installer/configmaps/windows-efi-installer-configmaps.yaml

```diff
@@ -250,7 +250,7 @@ data:
                     </OSImage>
                 </ImageInstall>
                 <UserData>
-                    <AcceptEula>true</AcceptEula>
+               <!-- <AcceptEula>true</AcceptEula> -->
```

Comment:
Would it make sense to set it to false instead of making it a comment?

Reply:
Both ways will work :) Your approach makes more sense